vm-setup/library/generate_macs.py

```diff
@@ -24,19 +24,19 @@
 version_added: "2.0"
 short_description: Generate a list of Ethernet MAC addresses
 description:
-   - Generate a list of Ethernet MAC addresses suitable for baremetal testing.
+   - Generate a list of Ethernet MAC addresses suitable for external testing.
 '''
 
 MAX_NUM_MACS = math.trunc(0xff / 2)
 
 
-def generate_baremetal_macs(nodes, networks):
-    """Generate an Ethernet MAC address suitable for baremetal testing."""
+def generate_external_macs(nodes, networks):
```

Comment:
to me macs are not related to external network would prefer to name it something related to VM `generate_baremetal_vm_macs`